jvm/src/test/scala/org/atnos/site/Installation.scala

```diff
@@ -70,6 +87,10 @@ You can also access all the syntax imports at once with:
 import org.atnos.eff.syntax.all._
 ```
 
+#### Intellij support
+
+Intellij doesn't support implicit-directed type inference. It'll work when you build it.
```

Comment:
I would like to avoid snarky comments here :-). Can you please reference issues like https://youtrack.jetbrains.com/issue/SCL-11140 or https://youtrack.jetbrains.com/issue/SCL-10753 instead?